Automattic/woocommerce-payments
# [Mobile Payments] Partially addresses display of card-present transactions

Partially addresses #1813

I drafted this PR to facilitate mobile testing. It is not expected this is the final iteration of this react component.

#### Changes proposed in this Pull Request

- Follows the pattern for giropay, etc and adds a new react component that can render details of a card-present transaction.

#### Testing instructions

* Use the WooCommerce iOS app to collect a payment for an order using a Stripe Point of Sale Test Card.
* Ensure you are able to access the details view for the transaction (before this PR, the screen would be mostly blank)
* Ensure that web based card payments continue to be displayed properly

#### What This Lacks

* There are various fields on the card_present response that we might want to include that this does not cover. This just gets it working using the fields common to web card payments.

<img width="1034" alt="cardpresent" src="https://user-images.githubusercontent.com/1595739/118557897-b5ad5380-b71a-11eb-9eec-77a07de5dd0d.png">

cc @laurendavissmith @ctarda 

-------------------
